app/components/header.tsx

```diff
@@ -1,16 +1,19 @@
 "use client";
 
 import { Percent, UsersIcon } from "lucide-react";
+import { ModeToggle } from "@/components/mode-toggle";
```

Comment:
I think `ThemeToggle` would be a much clearer name.
```suggestion
import { ThemeToggle } from "@/components/theme-toggle";
```